frontend/src/components/Checkbox/Checkbox.tsx

```diff
@@ -50,6 +54,7 @@ export const Checkbox = forwardRef<CheckboxProps, 'input'>(
         colorScheme={colorScheme}
         ref={ref}
         {...props}
+        {...(highContrast && { variant: 'highContrast' })}
```

Comment:
```suggestion
       {...(highContrast ? { variant: 'highContrast' } : {})}
```

nit to be consistent with `frontend/src/components/Button/Button.tsx`